src/Icon/Icon.tsx

```diff
@@ -1234,6 +1234,106 @@ export const Icon: FC<Props> = ({
         </svg>
       )
     }
+
+{
+      //LOCATION
+      render === 'location' && (
+        <svg
+          width="24"
+          height="24"
+          viewBox="0 0 24 24"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
```

Comment:
Can you please update all of them to have `width="100%" height="100%"` and remove `width="100%" height="100%"`.
Also location has an odd indentation of the top `{`